packages/client/src/orders/types/getReturnsFromOrder.types.ts

```diff
@@ -0,0 +1,4 @@
+import type { Config } from '../..';
+import type { Return } from '../../returns/types';
+
+export type GetOrderReturns = (id: string, config?: Config) => Promise<Return>;
```

Comment:
This should return an array of `Return`. Can you confirm it?

Reply:
✅